modules/acquia_cms_site_studio/acquia_cms_site_studio.info.yml

```diff
@@ -4,16 +4,16 @@ description: 'Module for Site Studio Installation Code.'
 core_version_requirement: ^9
 type: module
 dependencies:
-  - acquia:cohesion
-  - acquia:cohesion_base_styles
-  - acquia:cohesion_custom_styles
-  - acquia:cohesion_elements
-  - acquia:cohesion_style_guide
-  - acquia:cohesion_style_helpers
-  - acquia:cohesion_sync
-  - acquia:cohesion_templates
-  - acquia:cohesion_website_settings
-  - acquia:config_ignore
+  - drupal:cohesion
+  - drupal:cohesion_base_styles
+  - drupal:cohesion_custom_styles
+  - drupal:cohesion_elements
+  - drupal:cohesion_style_guide
+  - drupal:cohesion_style_helpers
+  - drupal:cohesion_sync
+  - drupal:cohesion_templates
```

Comment:
@vishalkhode1 - I verified the changes locally and its working fine, though I have one though.

Shouldn't cohesion module prefix with project:module_machine_name name like `acquia:cohesion` and similarly for all cohesion modules.

> [Let drupal know about your module with info.yml file](https://www.drupal.org/docs/creating-custom-modules/let-drupal-know-about-your-module-with-an-infoyml-file)
dependencies: a list of other modules your module depends on. Dependencies on Drupal core or contrib modules should be namespaced in the format {project}:{module}, where {project} is the project name as it appears in the Drupal.org URL (e.g. drupal.org/project/views) and {module} is the module's machine name. Dependencies can also include version restrictions, for examplewebform:webform (>=8.x-5.x). Note that if your module has dependencies on other contributed modules or libraries, these should be declared in the module's composer.json file. If you have local custom modules that are dependent on each other you can use {module}:{module} (or {module}:{submodule} for sub-modules.)